regolith/interact.py

```diff
@@ -0,0 +1,26 @@
+"""
+Loads the dbs for interactive sessions
+
+Usage
```

Comment:
Will this render correctly with automodule?

Reply:
Probably not, you want the following to be a code-block, right?